docs/saml-2-0/README.md

```diff
@@ -9,6 +9,12 @@ In your IdP you can provide the following endpoint URLs to configure SSO, where
 - Single Logout Service (SLS): `https://<site-url>/sso/sls`
 - Assertion Consumer Service (ACS): `https://<site-url>/sso/verify`
 
+**Note:**: `<site-url>` will default to your primary network URL rather than the current site's URL. For single site mode use the following filter:
+
+```php
+add_filter( 'wpsimplesaml_network_activated', '__return_false' );
```

Comment:
Given that the module code has 
```
add_filter( 'wpsimplesaml_network_activated', '__return_true' );
```
in its boostrap routine here https://github.com/humanmade/altis-sso/blob/907f255daa9e867ff7c5e1834a86c68df6daf2dd/inc/saml/namespace.php#L29C2-L29C66

Should this example have a priority > 10 to ensure it runs later?

Reply:
Ah nice catch. In all likelihood this custom code would run after the module but best to be sure!
```suggestion
add_filter( 'wpsimplesaml_network_activated', '__return_false', 100 );
```